README.md

```diff
@@ -50,20 +54,43 @@ This demo was built using [React](https://reactjs.org/), a framework for JavaScr
 git clone https://github.com/mindee/doctr-tfjs-demo.git
 ```
 
-Then install the project using the following command:
+Then install the project's dependencies using the following command:
 
 ```shell
 cd doctr-tfjs-demo
 yarn install
 ```
 
-### Running the app locally
+### Running the app
+
+#### Production mode
+
+Alternatively, if you are looking at a production situation, first build the bundle and serve it:
+```shell
+yarn build
+serve --no-clipboard -s build
+```
+then navigate to the URL with your favorite web browser
+
+#### Development mode
 
 Once all dependencies have been installed, launch the app using:
 ```shell
 yarn start
 ```
-and navigate with your web browser to the URL in the console. 
+and navigate with your web browser to the URL in the console.
+
+### Using Docker container
+
+Lucky for you, if you prefer working with containers, we provide a minimal Docker image. You can build it with:
```

Comment:
maybe we could have a little line saying the build can be kind of long to reassure users, WDYT ? 
(it took me around 3 minutes to build it locally)

Reply:
Good idea :+1: